float64 stamp
float64[6] er
float64[6] f
float64[6] fd
float64[6] f_ee
float64[6] f_filter
float64[6] f_s
uint8 status
